src/api/block.rs

```diff
@@ -278,7 +280,253 @@ fn tokenomics_transaction_operations(block_summary: &BlockSummary) -> Vec<Operat
+                        ))),
+                        coin_change:          None,
+                        metadata:             None,
+                    });
+                }
+                if transaction_fees.microccd != 0 {
+                    res.push(Operation {
+                        operation_identifier: Box::new(OperationIdentifier::new(next_index(
+                            &mut index_offset,
+                        ))),
+                        related_operations:   None,
+                        _type:                OPERATION_TYPE_BLOCK_ACCRUE_REWARD.to_string(),
+                        status:               Some(OPERATION_STATUS_OK.to_string()),
+                        account:              Some(Box::new(AccountIdentifier::new(format!(
+                            "{}{}",
+                            ACCOUNT_ACCRUE_POOL_PREFIX,
+                            baker_id.to_string()
+                        )))),
+                        amount:               Some(Box::new(amount_from_uccd(
+                            transaction_fees.microccd as i128,
+                        ))),
+                        coin_change:          None,
+                        metadata:             None,
+                    });
+                }
+                if passive_reward.microccd != 0 {
+                    res.push(Operation {
+                        operation_identifier: Box::new(OperationIdentifier::new(next_index(
+                            &mut index_offset,
+                        ))),
+                        related_operations:   None,
+                        _type:                OPERATION_TYPE_BLOCK_ACCRUE_REWARD.to_string(),
+                        status:               Some(OPERATION_STATUS_OK.to_string()),
+                        account:              Some(Box::new(AccountIdentifier::new(format!(
+                            "{}{}",
+                            ACCOUNT_ACCRUE_POOL_PREFIX, POOL_PASSIVE
+                        )))),
+                        amount:               Some(Box::new(amount_from_uccd(
+                            passive_reward.microccd as i128,
+                        ))),
+                        coin_change:          None,
+                        metadata:             None,
+                    });
+                }
+                if baker_reward.microccd != 0 {
+                    res.push(Operation {
+                        operation_identifier: Box::new(OperationIdentifier::new(next_index(
+                            &mut index_offset,
+                        ))),
+                        related_operations:   None,
+                        _type:                OPERATION_TYPE_BLOCK_ACCRUE_REWARD.to_string(),
+                        status:               Some(OPERATION_STATUS_OK.to_string()),
+                        account:              Some(Box::new(AccountIdentifier::new(
+                            ACCOUNT_REWARD_BAKING.to_string(),
+                        ))),
```

Comment:
```suggestion
                        account:              Some(Box::new(AccountIdentifier::new(format!(
                            "{}{}",
                            ACCOUNT_ACCRUE_POOL_PREFIX,
                            baker_id.to_string()
                        )))),
```
@td202 is this what you mean?